pontoon/base/templates/translate.html

```diff
@@ -174,6 +174,7 @@
             {{ Filter.item('untranslated', 'Untranslated') }}
             {{ Filter.item('unchanged', 'Unchanged') }}
             {{ Filter.item('has-suggestions', 'Unreviewed Suggestions') }}
+            {{ Filter.item('rejected', 'Rejected Suggestions') }}
```

Comment:
Good point. We also don't say *Approved Suggestions*.

I think we should still keep *Unreviewed Suggestions*, though. For easier distinction with *Suggested*. Which will get rid of as part of bug [1377969](https://bugzilla.mozilla.org/show_bug.cgi?id=1377969). And then we start calling them just *Suggestions*.